packages/dev-server/src/createServer.ts

```diff
@@ -91,6 +91,7 @@ export async function createServer(config: Server.Config) {
 
   instance.addHook('onSend', async (request, reply, payload) => {
     reply.header('X-Content-Type-Options', 'nosniff');
+    reply.header('X-React-Native-Project-Root', process.cwd());
```

Comment:
is this true for monorepos as well? if not, how we can make it configurable?

Reply:
> is process.cwd() correct for monorepos as well?

probably not the best, updated to use `rootDir` from `config` ✅

Reply:
`rootDir` looks good, let's leave it at that